content/200-concepts/150-data-platform/01-about-platform/100-platform-limits-and-status.mdx

```diff
@@ -0,0 +1,88 @@
+- multiple Prisma Clients connect that use a different version of the Prisma schema
+
+| Concurrency limits                                                                                         | Number                            |
+| ---------------------------------------------------------------------------------------------------------- | --------------------------------- |
+| Connection pool per proxy instance                                                                         | **Default**: 60<br />**Max**: 120 |
+| Maximum number of concurrent proxy instances on **Early Adopter** plan                                     | 3                                 |
+| Maximum number of concurrent proxy instances on **Free** plan                                              | 16                                |
+| Maximum number of concurrent proxy instances on **Scale** plan                                             | 16                                |
+| Maximum number of concurrent proxy instances on **Enterprise** plan                                        | Custom                            |
+| Approximate maximum queries per second with 16 proxy instances\*<br /><br />(**Free** and **Scale** plans) | ~19,200                           |
+
+<Admonition>
+
+**Note**<br /><br /> \* The number in the last row does not take into account the complexity of the queries.<br /><br />Complex queries which might require multiple database operations might not reach the maximum numbers and might slow down other queries waiting for a database connection.
+
+</Admonition>
+
+<Admonition>
+
+**Extending the concurrency limits**<br /><br />
+Contact [Support](/concepts/data-platform/contact-support) if your project requirements exceed the documented limits or your project is already hitting the current limits.
+
+</Admonition>
+
+## Compatibility with Prisma ORM
+
+We are dedicated to keeping the Prisma Data Platform compatible with the latest version of the Prisma ORM.
+
+- **Compatibility of the platform Data Browser, Query Console, Schema with the Prisma ORM**
+  - The interfaces are updated after each Prisma ORM release and aim to support the latest version.
+  - In rare occasions, there might be several days worth of a delay before the interfaces begin to support the latest version of the Prisma ORM.
+- **Compatibility of the Prisma Data Proxy with the Prisma ORM**
+  - Prisma Data Proxy is updated almost immediately to the latest version of the Prisma ORM.
+  - Prisma Data Proxy does not yet support the preview features `interactiveTransactions` and `metrics`.
+
+## Availability and SLOs
+
+The [Prisma Status](https://www.prisma-status.com/) page shows information about:
+
+- current status of the platform
+- current and historical uptime of the Prisma Data Proxy in all supported locations
+- current and historical uptime of the Platform Control Plane (Web interface and API)
```

Comment:
I agree this reads better, but "control plane" is being removed, and this bullet is specifically the web UI and API of the Control Plane, as opposed to the APIs of other parts of the PDP like Data Proxy.

I believe that's important to retain. Maybe Control Plane could go in parenthesis instead.

Reply:
@albertoperdomo if we can skip saying "control plane", it would be best because this is a little bit of the internal jargon that's difficult to translate externally.

Reply:
I agree it's jargon, but data proxy API is not included here and the proposal does not reflect this fact.
We also use Control Plane in the [status page are are linking to](https://www.prisma-status.com/).

Just to be on the same page:

![image](https://user-images.githubusercontent.com/24216/173619811-7d6849e8-2fbf-4c5b-981c-5589b7f158a1.png)